mmdeploy/__init__.py

```diff
@@ -2,7 +2,7 @@
 import importlib
 
 from mmdeploy.utils import get_root_logger
-from .version import __version__  # noqa F401
+from .version import *  # noqa F401
```

Comment:
could only import `__version__` and `version_info`

```

from .version import __version__, version_info # noqa F401
```

Reply:
in that case, in 'version. py' “__all__ = ['__version__', 'version_info', 'parse_version_info']”,What is the meaning of it?
Function "parse_version_info", The same error occurs with info.

Reply:
then, we could remove `parse_version_info` from `__all__`

Reply:
Thank you for your answer. If you don't use the feature of '__all__' in this way, this PR can be modified as you mentioned. Do you think it's okay?
`from .version import __version__, version_info # noqa F401`

Reply:
works for me.

Reply:
@xiabo123 hi, when will you update this PR?

Reply:
Has been updated